beast/physicsmodel/tests/test_stellar_prior_weights.py

```diff
@@ -1,14 +1,187 @@
 import numpy as np
 
-from beast.physicsmodel.prior_weights_stars import compute_distance_prior_weights
+from beast.physicsmodel.prior_weights_stars import *
```

Comment:
You need to explicitly import the functions you want to use.  Importing all with a `*` is discouraged as it means the flake8 tests can't check that all the functions used are actually present.

Reply:
got it thanks